mantle/cmd/ore/azure/upload-blob-arm.go

```diff
@@ -101,12 +101,15 @@ func runUploadBlobARM(cmd *cobra.Command, args []string) {
 		if err := api.UploadBlob(ubo.storageacct, *k.Value, ubo.vhd, ubo.container, ubo.blob, ubo.overwrite); err != nil {
 			plog.Fatalf("Uploading blob failed: %v", err)
 		}
-		break
```

Comment:
Hmm, I'm guessing that `break` was there on purpose so that it only used the first service key. Otherwise, we'd be uploading the same thing twice. Probably what we should instead do here to appease the linter is to just rework this to access the first key only and not use a loop at all.

/cc @arithx to confirm.

Reply:
Yes I believe that is the intent.

Reply:
Thank you for review and kindly reminding. Updated.